src/main/resources/templates/course.twig

```diff
@@ -6,65 +6,72 @@
 {% endblock %}
 
 {% block content %}
+    <input type="hidden" id="courseID" name="courseID" value="{{ course.id }}"/>
     <div class="course-editor-body">
         <div class="course-editor-title">
-            <h1>{{ course.name }}</h1>
+            <h1>{{ course.name }} <span style="color: grey;font-size:smaller;">{{ course.section }}</span></h1>
         </div>
         <div class="course-editor-grid">
             <div id="course-seating-grid"></div>
         </div>
         <div class="course-editor-controls">
             <button id="editsave" type="button" class="btn btn-primary">Edit</button>
+            <button id="editclearseating" type="button" class="btn btn-primary">Clear Student Seating</button>
+            <button id="editcleartemplate" type="button" class="btn btn-primary">Reset Seat Template</button>
+        </div>
+    </div>
+    <!--Clear Student Seating Warning Modal-->
+    <div class="modal fade" id="modal-danger-clear-student" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+        <div class="modal-dialog modal-notify modal-danger" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <p class="heading lead">Clear Student Seating</p>
+                    <button type="button" id=modal-danger-clear-student-close-top" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true" class="white-text">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    <div class="text-center">
+                        <i class="fa fa-check fa-4x mb-3 animated rotateIn"></i>
+                        <p>Are you sure you wish to clear student seating? This will remove students from their assigned seating.
+                        The grid dimensions and seat states will be preserved.</p>
+                    </div>
+                </div>
+                <div class="modal-footer justify-content-center">
+                    <a type="button" id="modal-danger-clear-student-continue" class="btn btn-primary-modal">Continue<i class="fa fa-diamond ml-1"></i></a>
+                    <a type="button" id="modal-danger-clear-student-close-bottom" class="btn btn-outline-secondary-modal waves-effect" data-dismiss="modal">Take me back!</a>
+                </div>
+            </div>
+        </div>
+    </div>
+    <!--Clear Student Seating Warning Modal-->
```

Comment:
Should this comment say `Reset Template Warning Modal` instead?

Reply:
You’re right Mack, thanks for finding this